Examples/Run/Common/src/CommonOptions.cpp

```diff
@@ -41,9 +41,9 @@ void ActsExamples::Options::addSequencerOptions(
                     "The number of events to process. If not given, all "
                     "available events will be processed.")(
       "skip", value<size_t>()->default_value(0),
-      "The number of events to skip")(
-      "jobs,j", value<int>()->default_value(-1),
-      "Number of parallel jobs, negative for automatic.");
+      "The number of events to skip")("jobs,j", value<int>()->default_value(-1),
+                                      "Number of parallel jobs, negative for "
+                                      "automatic.");
```

Comment:
Pure formatting change right?

Reply:
Yes, I have a newer clang-format version locally (and don't run the docker for the moment), so it got changed and then via lxplus changed again.